c_plus_plus/src/model/recorder/recorder.h

```diff
@@ -0,0 +1,38 @@
+#ifndef RECORDER_H
+#define RECORDER_H
+
+#include "i_recorder.h"
+
+class QMediaRecorder;
+class QCamera;
+class QCameraInfo;
+class QCameraViewfinder;
+
+namespace Model
+{
+
+class Recorder : public IRecorder
+{
+public:
+    explicit Recorder(const QString cameraDevice, QWidget *parent = nullptr);
+    void start(const QString outputPath) override;
+    void stop() override;
```

Comment:
I think we should have a destructor that stop the camera if it's not already stopped.